### PR TITLE
NO-JIRA Chart refactoring

### PR DESCRIPTION
**Pull request by davividal**
Mirroring https://github.com/SonarSource/helm-chart-sonarqube/pull/473 due to Cirrus CI limitations.

### CODE REVIEW

**Review by carminevassallo — APPROVED** [review]
The PR was reviewed and accepted by @davividal already. Approving it...

**Comment by davividal** [comment]
Again, thanks, @bonddim, for your fantastic contribution!

File: charts/sonarqube/templates/_pod.tpl [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+          subPath: data
+      env:
+        - name: http_proxy
+          value: {{ default "" .Values.prometheusExporter.httpProxy }}
+        - name: https_proxy
+          value: {{ default "" .Values.prometheusExporter.httpsProxy }}
+        - name: no_proxy
+          value: {{ default "" .Values.prometheusExporter.noProxy }}
+        {{- range (include "sonarqube.combined_env" . | fromJsonArray)  }}
+        - name: {{ .name }}
```
Comment (jonesbusy) [inline charts/sonarqube/templates/_pod.tpl:164]: This break again from the 10.5.1 and secret values

Comment (bonddim) [reply charts/sonarqube/templates/_pod.tpl:164]: @jonesbusy , correct I missed this change during rebase with master branch.

Comment (bonddim) [reply charts/sonarqube/templates/_pod.tpl:164]: @jCOTINEAU , pls, pay attention to this thread. 
Some changes were missed from original PR.

Comment (jCOTINEAU) [reply charts/sonarqube/templates/_pod.tpl:164]: @davividal is working on it, we are discussing internally and will notify you soon

**Pull request merged**